tests/test_tablib_dbfpy_packages_utils.py

```diff
@@ -151,7 +151,7 @@ def test_getDateTime_datetime_string(self):
 
         # Act / Assert
         with self.assertRaises(NotImplementedError):
-            output = utils.getDateTime(value)
+            _ = utils.getDateTime(value)
```

Comment:
Nope, fine like this:
```suggestion
            utils.getDateTime(value)
```

Reply:
Thanks.